src/app/backend/handler/apihandler.go

```diff
@@ -220,22 +243,22 @@ func CreateHTTPAPIHandler(client *clientK8s.Clientset, heapsterClient client.Hea
 
 	apiV1Ws.Route(
 		apiV1Ws.POST("/appdeployment").
-			To(apiHandler.handleDeploy).
+			To(apiHandler.injectApiClient(apiHandler.handleDeploy)).
```

Comment:
Agreed. This should be "forward and require Bearer token for all requests, ignore Bearer for hand-crafted list of request".